README.md

```diff
@@ -1,47 +1,42 @@
 # **Portshift Orb for CircleCi**
 
-## **What is Portshift Orb?**
-With this orb you can easily integrate Portshift identity creation and vulnerabilities scans into your CircleCI Workflows.
-Portshift identity creation is essential part of Portshift runtime security, authenticating and authorizing containers build and deployments with vulnerabilities scan results. The scan engine is currently based on [Clair](https://github.com/quay/clair), in the future additional scan engines will be added to Portshift Orb to enhance the vulnerabilities information. 
-Portshift Orb is an easy way to get started with Portshift runtime security platform, the image identity and the vulnerabilities scan results will be shown in Portshift Management platform.
+## **What is the Portshift Orb?**
+The Portshift Orb enables you to you to scan images for vulnerabilities in CircleCI, and report results to the Portshift server.
+With this orb you can easily integrate Portshift vulnerabilities scans into your CircleCI Workflows.
 
-## **How to use Portshift Orb**
+Portshift workload runtime security policies can use vulnerabilities scan results as a criteria for allow or blocking images from running in Portshift protected environments.
 
-### **Prerequisites**
-1. User-Account: Create a user account in portshift (note: Portshift is a s commercial software, so you'll need a license to create a user accouunt before using this orb Go to: [www.portshift.io](www.portshift.io). After receiving a "New User" onboarding email, go to Portshift's user-portal (www.console.portshift.io) to enter your domain.
-
-2. Define Service Account: In the user domain you need to define a new "service account":
+The scan engine is based on [Clair](https://github.com/quay/clair), though additional scan engines may be added, to enhance the vulnerabilities information. 
```

Comment:
David,
why adding the information at this stage?

Reply:
this was in the original text of the README (from Tomer). You prefer to drop it?